packages/dds/sequence/src/defaultMap.ts

```diff
@@ -262,9 +257,12 @@ export class DefaultMap<T extends IntervalCollection<any> = IntervalCollection<a
 
 	/**
 	 * Populate the kernel with the given map data.
-	 * @param data - A JSON string containing serialized map data
+	 *
+	 * @param serialized - A JSON string containing serialized map data
 	 */
-	public populateFromSerializable(json: IMapDataObjectSerializable): void {
+	public populate(serialized: string): void {
+		const json = this.serializer.parse(serialized) as IMapDataObjectSerializable;
```

Comment:
Let's call this somethign else... it has real handles in it... that's not JSON-like.